base/doc/ltnews38.tex

```diff
@@ -283,29 +410,80 @@ \subsection{In the programming layer}
  >\fp_eval:n{dinner(duck)}
 $
 \ExplSyntaxOff
-\end{document}
 \end{verbatim}
 
+The computation above would then generate
+\begin{quote}
+\ExplSyntaxOn
+\fp_new_variable:n{duck}
+\fp_new_function:n{dinner}
+\fp_set_function:nnn{dinner}{duck}
+                    {duck - 0.25 * duck}
+\fp_set_variable:nn{duck}{1}
+$\fp_eval:n{duck}
+ >\fp_eval:n{dinner(duck)}
+  \fp_set_variable:nn{duck}{dinner(duck)}
+ >\fp_eval:n{dinner(duck)}
+  \fp_set_variable:nn{duck}{dinner(duck)}
+ >\fp_eval:n{dinner(duck)}
+  \fp_set_variable:nn{duck}{dinner(duck)}
+ >\fp_eval:n{dinner(duck)}
+$
+\ExplSyntaxOff
+\end{quote}
+
 Users will be able to access added functions without needing to use the
 \pkg{expl3} layer. It is possible that a future release of \LaTeX{} will
 add the ability to create and set floating point variables to the
 document level: this will be examined based on feedback on the
 utility of the programming layer change.
 
-\section{Bug fixes}
 
-\section{Changes to packages in the \pkg{amsmath} category}
+\section{\emph{Removed} commands}
 
-\section{Changes to packages in the \pkg{graphics} category}
+It is very rare that commands are removed from the \LaTeX{} kernel. However, in
+this release we have elected to remove \cs{GetDocumentCommandArgSpec},
+\cs{GetDocumentEnvironmentArgSpec}, \cs{ShowDocumentCommandArgSpec} and
+\cs{ShowDocumentEnvironmentArgSpec} from the kernel. These commands have been
+moved back to the \enquote{stub} \pkg{xparse} provided in \pkg{l3packages}. The
+reason for this change is that the commands were essentially part of debugging
+early forms of the kernel code, and exposed implementation details in a way that
+was not helpful.
```

Comment:
If what that "and" conjuncts are
- "the commands were essentially part of debugging early forms of the kernel code" and
- "exposed implementation details in a way that was not helpful",

then maybe

```suggestion
moved back to the \enquote{stub} \pkg{xparse} provided in \pkg{l3packages}. The
reasons for this change are that the commands were essentially part of debugging
early forms of the kernel code, and exposing implementation details in a way
like that was not helpful.
```

Reply:
I think all it need is taking the comma out, but I leave this to @josephwright

Reply:
Need to know the original intent.

Reply:
I feel the comma is right: one for Barbara as a proper copy-editor

Reply:
The version with the two blobs is far clearer. 
Why not go with this?

Reply:
@josephwright No copy-editor, however proper, can edit text that is so ambiguous.  
Maybe a clairvoyant copy-editor is needed?

Reply:
Just for the record, this is further resolved/improved in commit 6ec65ac0 (Re-write a paragraph, 2023-10-26).